.github/workflows/triage_closed_issue_comment.yml

```diff
@@ -6,43 +6,46 @@ on:
       ADD_TO_TRIAGE_BOARD_TOKEN:
         required: true
   issue_comment:
-    types:
-      - created
+    types: [created]
+    issues:
+      types: [closed]
 jobs:
   move-to-new-issue-status:
-    if: |
-      !github.event.issue.pull_request &&
-      github.event.issue.state == 'closed' &&
-      github.event.comment.created_at != github.event.issue.closed_at &&
-      github.event.sender.login != 'cypress-bot[bot]'
     runs-on: ubuntu-latest
     steps:
-      - name: Get project data
-        env:
-          GITHUB_TOKEN: ${{ secrets.ADD_TO_TRIAGE_BOARD_TOKEN }}
-          ORGANIZATION: 'cypress-io'
-          REPOSITORY: ${{ github.event.repository.name }}
-          PROJECT_NUMBER: 9
-          ISSUE_NUMBER: ${{ github.event.issue.number }}
-        run: |
-          gh api graphql -f query='
-            query($org: String!, $repo: String!, $project: Int!, $issue: Int!) {
-              organization(login: $org) {
-                repository(name: $repo) {
-                  issue(number: $issue) {
-                    projectItems(first: 10, includeArchived: false) {
-                      nodes {
-                        id
-                        fieldValueByName(name: "Status") {
-                          ... on ProjectV2ItemFieldSingleSelectValue {
-                            name
-                            field {
-                              ... on ProjectV2SingleSelectField {
-                                project {
-                                  ... on ProjectV2 {
-                                    id
-                                    number
-                                  }
+    - name: Make sure comment was added more than 20 seconds after issue closed
+      run: |
+        time_diff=$(( ($(date +%s -d "${{ github.event.comment.created_at }}" ) - $(date +%s -d "${{ github.event.issue.closed_at }}" )) > 20))
+        echo 'COMMENT_ADDED_AFTER_CLOSE='$time_diff >> $GITHUB_ENV
+    if: |
+      env.COMMENT_ADDED_AFTER_CLOSE == '1' && 
+      github.event.sender.login != 'cypress-bot[bot]' &&
+      github.event.sender.login != 'github-actions[bot]' 
```

Comment:
I'm surprised this doesn't need to be indented after line 24. It seems like it's in-between two steps, which I guess works??